tests/core/RequestHandler.spec.ts

```diff
@@ -139,39 +138,43 @@ describe('RequestHandler', () => {
 
   it('should resolve LEGACY long form did from test vectors correctly', async () => {
```

Comment:
Legacy long form cannot switch to camel case. There are clients of ION that are dependent on it not breaking until they fully switch over to the new form. The legacy code path is separated out so we can duplicate functions and add TODO to delete them once everyone migrates over.